HospitalBeds/R4/resources/Quesionnaire-R4-HospitalBeds.json

```diff
@@ -142,7 +142,6 @@
     },
```

Comment:
Item 1 should also have text.
1.4 and 1.5 are not required
1.5 should be answerOption